web/src/components/domains/task/TaskItem/TaskItem.jsx

```diff
@@ -1,11 +1,56 @@
 import styles from './TaskItem.module.css';
-
+import { PropTypes } from 'prop-types';
 /*
 Please create the <TaskItem /> component following the design from the Figma file.
 Please make sure to add styles using CSS Modules.
 Add the necessary props to the component.
 */
 
-export function TaskItem(props) {
-  return <div className={styles.itemWrapper}></div>;
+function TaskItem(props) {
+  const colorState = () => {
+    if (props.ProjectStatus === 'Low') {
+      return styles.buttonGreen;
+    }
+    if (props.ProjectStatus === 'Medium') {
+      return styles.buttonOrange;
+    }
+    if (props.ProjectStatus === 'High') {
+      return styles.buttonRed;
+    }
+  };
+
+  return (
+    <div className={styles.mainContainer}>
+      <div className={styles.container}>
+        <h4 className={styles.Task}>{props.ProjectTask}</h4>
+        <button className={colorState()}>{props.ProjectStatus}</button>
```

Comment:
The status element it is a text badge and is not interactive, so could be better to be replaced by a div or span tag.
```suggestion
        <div className={colorState()}>{props.ProjectStatus}</div>
```